src/webportal/src/app/job/job-view/fabric/job-detail.jsx

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import classNames from 'classnames';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {initializeIcons} from '@uifabric/icons';
+import {FontClassNames} from '@uifabric/styling';
+
+import t from './tachyons.css';
+
+import Top from './job-detail/components/top';
+import Summary from './job-detail/components/summary';
+import {SpinnerLoading} from './job-detail/components/loading';
+import TaskRole from './job-detail/components/task-role';
+import {fetchJobConfig, fetchJobInfo, fetchSshInfo, stopJob} from './job-detail/conn';
+import {getHumanizedJobStateString} from './job-detail/util';
+
+initializeIcons();
+
+class JobDetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      jobInfo: null,
+      jobConfig: null,
+      loading: true,
+      sshInfo: null,
+    };
+    this.stop = this.stop.bind(this);
+  }
+
+  componentDidMount() {
+    void this.reload();
+  }
+
+  async stop() {
+    await stopJob();
+    await this.reload();
+  }
+
+  async reload() {
+    await Promise.all([
+      fetchJobInfo().catch(alert),
+      fetchJobConfig().catch((err) => {
+        // console.log(err);
```

Comment:
remove?

Reply:
Maybe I should add some comments here instead.

Reply:
Add more error handling logic